addons/website/views/snippets/s_website_form.xml

```diff
@@ -8,7 +8,7 @@
                 <div class="s_website_form_rows row s_col_no_bgcolor">
                     <div class="form-group col-12 s_website_form_submit" data-name="Submit Button">
                         <div style="width: 200px;" class="s_website_form_label"/>
-                        <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send">Submit</a>
+                        <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send o_submit oe_unremovable">Submit</a>
                         <span id="s_website_form_result"></span>
                     </div>
                 </div>
```

Comment:
Not sure if this is the best way to fix it, but with this class, backspacing on a button doesn't delete it anymore. I could also add o_submit class and a bunch of others to [isUnremovable](https://github.com/odoo/odoo/blob/c3528f268dfb498a7806cb9b6f4aa3d964f421ea/addons/web_editor/static/lib/odoo-editor/src/utils/utils.js#L931) to make sure we cannot remove submit buttons.